library/src/main/res/layout/activity_image_details.xml

```diff
@@ -1,12 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="#000000"
-    android:orientation="vertical">
+    android:background="@android:color/black">
+
+    <android.support.v7.widget.CardView
+        android:id="@+id/card_toolbar"
```

Comment:
using camelCase instead snake_case. 

I have plan to remove all 'kotlin android extensions' code to avoid memory issues that can occur while caching views. but it will take some time, so use until then, but I recommend using camelCase just like the existing conventions.